File: Source/BuildLogHandler/LogWriter.cpp
/*---------------------------------------------------------------------------------------------
 *  Copyright (c) Dolittle. All rights reserved.
 *  Licensed under the MIT License. See LICENSE in the project root for license information.
 *--------------------------------------------------------------------------------------------*/
#include <stdio.h>
#include <string>
using namespace std;

#include "LogWriter.h"

LogWriter::LogWriter(Config config)
{
    _config = config;

    //auto path << "blah";

    //auto path = config.Path+"/"+config.StepNumber+".log";

    //_file = fopen(path.)
}

void LogWriter::Write(std::string line)
{

    printf("%s", line.c_str());
}

File: Source/BuildLogHandlerGo/Output/LogWriter.go
/*---------------------------------------------------------------------------------------------
 *  Copyright (c) Dolittle. All rights reserved.
 *  Licensed under the MIT License. See LICENSE in the project root for license information.
 *--------------------------------------------------------------------------------------------*/
package Output

import (
	"github.com/dolittle-platform/continuous_improvement/Source/BuildLogHandlerGo/Config"
)

type LogWriter interface {
	Configure(config Config.Config)
	Write(line string)
}
